test/modules/placeholder-widget/index.js

```diff
@@ -2,7 +2,7 @@ module.exports = {
   extend: '@apostrophecms/widget-type',
   options: {
     label: 'Placeholder Test Widget',
-    initialModal: false
+    placeholder: true
```

Comment:
no impact on the tests because this options is used by Vue on front-end.
but does no harm to add it for clarity